manifests/application.pvn.yaml

```diff
@@ -4,17 +4,24 @@ application:
     slack:
       channel: audit
   releaseChannels:
-    - name: canary
+    - name: staging
       constants:
         - name: name
           string:
-            value: canary
+            value: staging
+      runtimes:
+        - name: config-raw
+          runtime: govalctl-config-raw
+          type: EXTENSION
```

Comment:
what's the status of staging? i thought we're only doing prerelease for now?

Reply:
Staging still exists but is mostly just for staging infra changes, we don't run repls there. Eventually, we want an end-to-end staging environment that is separate from prod. But web isn't able to support multiple environments yet.